Packs/DemistoRESTAPI/Integrations/CoreRESTAPI/CoreRESTAPI.js

```diff
@@ -3,6 +3,33 @@ if (serverURL.slice(-1) === '/') {
     serverURL = serverURL.slice(0,-1);
 }
 serverURL = serverURL + '/xsoar'
+var marketplace_url = params.marketplace_url? params.marketplace_url : 'https://storage.googleapis.com/marketplace-dist/content/packs/'
```

Comment:
this url should differ if the integration running in XSIAM or XSOAR

Reply:
There is no way to distinguish between XSIAM and XSOAR. It should be added manually by the customer.